system-tests/src/test/java/org/eclipse/edc/samples/transfer/streaming/Streaming02KafkaToHttpTest.java

```diff
@@ -88,12 +90,10 @@ public class Streaming02KafkaToHttpTest {
             ":transfer:streaming:streaming-02-kafka-to-http:streaming-02-runtime"
     ).configurationProvider(fromPropertiesFile(SAMPLE_FOLDER + "/streaming-02-runtime/consumer.properties")));
 
-    private final int httpReceiverPort = Ports.getFreePort();
-    private final MockWebServer consumerReceiverServer = new MockWebServer();
 
-    @BeforeEach
-    void setUp() throws IOException {
-        consumerReceiverServer.start(httpReceiverPort);
+    @BeforeAll
+    static void setUp() {
+        httpRequestLoggerContainer.start();
```

Comment:
this is not needed because the `@Container` annotation has been added to the logger container instance

Reply:
Noted and removed 🚀